### PR TITLE
Add enrollment attestation translations

### PR DESCRIPTION
**Pull request by anusmitaray**

### CODE REVIEW

File: attestation-translations/attestation_ar.txt
Comment (rowan-m) [inline attestation-translations/attestation_ar.txt:1]: Remove blank lines

File: attestation-translations/attestation_fa.txt
Comment (rowan-m) [inline attestation-translations/attestation_fa.txt:1]: Remove blank lines

File: attestation-translations/attestation_hu.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Az igazoló szervezet kijelenti, hogy nem használja a Privacy Sandbox API-kat vagy szolgáltatásokat arra a célra, hogy különböző oldalakon vagy alkalmazásokban megtudja, hogy Ön ugyanaz a felhasználó, és hogy más módon nem kerüli meg 
+a Privacy Sandbox adatvédelmét.
```
Comment (rowan-m) [inline attestation-translations/attestation_hu.txt:2]: Remove line break

File: attestation-translations/attestation_iw.txt
Comment (rowan-m) [inline attestation-translations/attestation_iw.txt:1]: Remove blank lines

File: attestation-translations/attestation_ru.txt
Comment (rowan-m) [inline attestation-translations/attestation_ru.txt:1]: Remove line break

File: attestation-translations/attestation_sr.txt
Comment (rowan-m) [inline attestation-translations/attestation_sr.txt:1]: Remove blank lines

File: attestation-translations/attestation_zh-TW.txt
Comment (rowan-m) [inline attestation-translations/attestation_zh-TW.txt:1]: Remove blank lines

**Review by rowan-m — CHANGES_REQUESTED** [review]

File: attestation-translations/attestation_ar.txt
Comment (anusmitaray) [reply attestation-translations/attestation_ar.txt:1]: Done

File: attestation-translations/attestation_fa.txt
Comment (anusmitaray) [reply attestation-translations/attestation_fa.txt:1]: Done

File: attestation-translations/attestation_hu.txt (hunk 1 above)
Comment (anusmitaray) [reply attestation-translations/attestation_hu.txt]: Done

File: attestation-translations/attestation_ru.txt
Comment (anusmitaray) [reply attestation-translations/attestation_ru.txt:1]: Done

File: attestation-translations/attestation_sr.txt
Comment (anusmitaray) [reply attestation-translations/attestation_sr.txt:1]: Done

**Review by rowan-m — APPROVED** [review]
LGTM, but hold on merging for final confirmation from @GeorgiaFranklin

**Pull request merged**